File: Summation_Nonfactors/Sum.java
/*
    Author : Pakshal Jain
    Date : 05/10/2022
    Program : To Calculate Summation of Nonfactors of Entered Number
*/

//Required Libraries
import java.lang.*;
import java.util.*;

//Class Declaration
class NonFact 
{
    //Characteristics of Class
    int No = 0;
    
    //Constructor of Class
    public NonFact(int no)
    {
        this.No = no;
    }
    //Behaviour of Class
    int summation()
    {
        int i = 0;
        int ans = 0;

        //Logic
        for(i = 1;i < this.No;i++)
        {
            if(this.No % i != 0)
            {
                ans = ans + i;        
            }
        }

        return ans;
    }
}

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

class Sum
{
    public static void main(String args[])
    {
        int No = 0;
        int iret = 0;

        Scanner sobj = new Scanner(System.in);
        System.out.println("Jay Ganesh....");
        
        System.out.println("Enter one Number");
        No = sobj.nextInt();

        //Creating object of Nonfact Class
        NonFact nobj = new NonFact(No);
        //Calling Behaviour of Class
        iret = nobj.summation();

        System.out.println("Summation of Nonfactors of Entered Number is : "+iret); 
    }    
}

File: Summation_Nonfactors/Sum.c
/*
    Auhtor : Pakshal Jain
    Date : 05/10/2022
    Program : To Find Summation of Nonfactors of Entered Number
*/

///////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Required Libraries
#include<stdio.h>

//Function To Calulcation Summation
int Summation(int No)
{
    int i = 0;
    int Sum = 0;    

    //Logic
    for(i = 1;i < No;i++)
    {
        if(No % i != 0)
        {
            Sum = Sum + i;
        }
    }

    return Sum;
}

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

int main() 
{
    int No = 0;
    int iret = 0;

    printf("Jay Ganesh....\n");
    
    printf("Enter One Number\n");
    scanf("%d",&No);

    //Call To Summation Function   
    iret = Summation(No);
    
    printf("Summation of Nonfactors of Entered Number is : %d",iret);

    return 0;
}